### ISSUE TITLE
Forge 1492 for Mc 1.7.10 and CoFH Core 308

### ISSUE THREAD

**Issue by Dream-Master** [CoFH/Feedback#835]
I using Forge 1492 and
CoFHCore-[1.7.10]3.0.3B2-294
every time i logged in the game I get the message to update Cofh Core.
than i try out this Version
CoFHCore-[1.7.10]3.0.4B1-308
After starting the game i get this error and the game stops loading
[17:35:15] [Client thread/ERROR] [FML/]: The mod CoFHCore (CoFH Core) requires mod versions [Forge@[10.13.3.1448,10.14)] to be available
I think there is something wrong with Forge 1448 and dependencies. I get the same error when i add the erebus mod(have a 1448 dependency) If i change this dependency to maybe 1360 i can start the game without any error. I guess it is here the same problem. I try to fork your project already but when i want to compiled it back i get so much errors because missing api or/and additional jar files in the libs folder.

Please can you fixe it.

Thanks

If you need more infos about my pack please contact me.

**Comment by KingLemming** [comment]
That version of Core is flagged as an alpha build. Don't use it.

Use only 3.0.3, the release version.

**Comment by Dream-Master** [comment]
Same with 3.03

**Comment by Dream-Master** [comment]
Can send you the fml file if you need it.

**Comment by Dream-Master** [comment]
[18:20:39] [Client thread/ERROR] [FML/]: The mod CoFHCore (CoFH Core) requires mod versions [Forge@[10.13.3.1448,10.14)] to be available

[18:20:39] [Client thread/DEBUG] [FML/]: 	CoFHCore(CoFH Core:1.7.10R3.0.3): CoFHCore-[1.7.10]3.0.3-303.jar (required-after:Forge@[10.13.3.1448,10.14);)

**Comment by KingLemming** [comment]
Sorry, dunno what to tell you. Can't reproduce. Forge 1448 works.

**Comment by Dream-Master** [comment]
Any guess where i can digging to find the error?

**Comment by GoetzLiedtke** [comment]
I am having the same problem as Dream-Master.  CoFHCore asks for 10.13.3.1448 but there is no Forge version of that number.  There is 10.13.4.1448  which does not satisfy the requirement and results in the same error.  I used the latest 1.7.10 recommended version of Forge (10.13.4.1492) which results in the error.  I tried Forge 10.13.3.1428, the last of the three series, and that resulted in the same error.  I'm currently looking through the CoFHCore code from GitHub to find where and how that dependency is set.  When I do, I will attempt to change it from 10.13.3.1448 to 10.13.4.1448, build the library, test it in my Minecraft mods folder and see if the error goes away.  If it does, I will send you the code.

**Comment by KingLemming** [comment]
10.13.4.1448 will satisfy the dep string as it is written. It works with 1448, truly.

**Comment by KingLemming** [comment]
Just incase you don't believe me for some odd reason:

![1492cofh](https://cloud.githubusercontent.com/assets/2118598/9834884/ec42f99a-599c-11e5-9cb6-31c399a45bdb.png)

**Comment by GoetzLiedtke** [comment]
I went back through the Forge log from the beginning and the version of
Forge after install of 1492 was still the old version.  I had to edit the
profile in the Minecraft launcher to use the newly installed version.
Silly me, expecting that an install would actually install :-)

Now back to debugging which mod is causing single digit FPS since the
latest java update.

Thanks for your patience.

Goetzliedtke@yahoo
